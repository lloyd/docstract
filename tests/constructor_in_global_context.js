/** And you certainly can't have a constructor in the
 *  *global* context! */

/** @constructor that builds things */
