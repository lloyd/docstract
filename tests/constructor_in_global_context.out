{
  "args": [
    "@constructor not allowed in global context"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>", 
  "stack": [
    [
      "<module>", 
      "got = ds.extractFromFile(os.path.join(testDir, test + \".js\"))"
    ], 
    [
      "extractFromFile", 
      "data = self.extract(contents)"
    ], 
    [
      "extract", 
      "self._analyzeBlock(block, context, firstBlock, data)"
    ], 
    [
      "_analyzeBlock", 
      "thisContext))"
    ]
  ]
}
